File: FitRackBackend/src/main/java/com/tericcabrel/authapi/entities/Statistics.java
package com.tericcabrel.authapi.entities;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "statistics")
@Data
public class Statistics {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;

    @Column(nullable = false)
    private int gamesPlayed;

    @Column(nullable = false)
    private int pointsScored;

    @Column(nullable = false)
    private int assists;

    @Column(nullable = false)
    private int rebounds;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "user_id", referencedColumnName = "id", nullable = false)
    private User user;
}

File: FitrackFrontend/src/app/services/statistics.ts

  export class Statistics {
    id: number;
    gamesPlayed: number;
    pointsScored: number;
    assists: number;
    rebounds: number;
    
  
    constructor(id: number, gamesPlayed: number, pointsScored: number, assists: number, rebounds: number) {
      this.id = id;
      this.gamesPlayed = gamesPlayed;
      this.pointsScored = pointsScored;
      this.assists = assists;
      this.rebounds = rebounds;
    }
  }